src/SegmentInterface.php

```diff
@@ -32,6 +32,17 @@ public function get($key, $alt = null);
 
     /**
      *
+     * Returns array of all key/value pairs in the segment.
+     *
+     * @param mixed $alt An alternative value to return if the segment name is not set.
+     *
```

Comment:
Don't change interface. Things will break. In that case we may need to go with 3.x .